src/daos-serialize/daos-serialize.c

```diff
@@ -0,0 +1,226 @@
+#include <stdio.h>
+#include <getopt.h>
+#include <string.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <inttypes.h>
+#include <unistd.h>
+#include <errno.h>
+#include <stdbool.h>
+
+/* for daos */
+#ifdef DAOS_SUPPORT
+#include "mfu_daos.h"
+#endif
```

Comment:
@dsikich Similar to `daos-serialize`, maybe we don't need the `#ifdef DAOS_SUPPORT`